firefox_ui_tests/remote/security/test_dv_certificate.py

```diff
@@ -0,0 +1,106 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestDVCertificate(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.url = "https://ssl-dv.mozqa.com"
+        self.locationbar = self.browser.navbar.locationbar
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+        self.page_info_window = None
+
+    def tearDown(self):
+        self.marionette.set_context("chrome")
+        try:
+            if self.page_info_window is not None:
+                self.page_info_window.close()
```

Comment:
Well, we could even remove those two lines given that you call `self.windows.close_all()` right below.